README.md

```diff
@@ -52,6 +52,15 @@ Install the following extensions:
 
 ## Code scaffolding
 
+## Commit format
+  We are using format by release and tags: https://semver.org/ 
+  ### Sumary
+  - fix is equal to Patch Release example: 1.0.1
+  - feat is equal to Feature Release example: 1.1.0
+  - perf is equal to Breaking Release example: 2.0.0
```

Comment:
Let's mark the keywords in bold:

  - **fix** is equal to Patch Release example: 1.0.1
  - **feat** is equal to Feature Release example: 1.1.0
  - **perf** is equal to Breaking Release example: 2.0.0